lib/telegram/bot/version.rb

```diff
@@ -1,6 +1,6 @@
 module Telegram
   module Bot
-    VERSION = '0.15.7'.freeze
+    VERSION = '0.15.8'.freeze
```

Comment:
Please keep it unchanged and add changelog line instead.

Reply:
Done